content/docs/concepts/asm/_index.md

```diff
@@ -0,0 +1,276 @@
+
+---
+title: "Auton Stabilization Mechanism (ASM)"
+linkTitle: "Auton Stabilization Mechanism (ASM)"
+weight: 10
+description: >
+  Auton Stabilization Mechanism: elements, the functions they perform, and the lifecycle for Auton and Newton supply.
+---
+
+## Overview
+
+This section describes the [Auton Stabilization Mechanism (ASM)](/glossary/#asm) and protocol. It details the elements comprising the mechanism, the functions the mechanism provides to compute and maintain a stable price for [Auton (ATN)](/glossary/#auton), and the lifecycle for Auton and [Newton (NTN)](/glossary/#newton) supply.
+
+For Auton stabilization control Autonity implements a [CDP](/glossary/#cdp)-based stabilization mechanism. Users take out CDP's, depositing Collateral Token (NTN) to borrow Auton at interest. As CDP's are taken out and repaid Collateral Token (NTN) and ATN are removed and returned to circulation, bringing equilibrium to supply and demand.
+
+CDP are maintained according to collateralization and liquidation ratios that set collateral amount and collateral value thresholds to keep a CDP in good health.  Auton is minted and burned as CDP's pass through their lifecycle, i.e. are taken out, repaid, withdrawn, and liquidated.
```

Comment:
@cmjc I don't see any problem with your description, because the collateral amount and collateral value thresholds are health factors that relate to liquidation ratio. However, IMO debt to collateral ratio may be a more direct description.

Reply:
Done in commit 6240f03709feec9b14e779a078f0624f50b1aa7d